x-pack/plugins/synthetics/public/apps/synthetics/state/overview/index.ts

```diff
@@ -58,7 +58,6 @@ export const monitorOverviewReducer = createReducer(initialState, (builder) => {
         ...state.pageState,
         ...action.payload,
       };
-      state.loaded = false;
```

Comment:
Since `fetchMonitorOverviewAction.get` is called when `setOverviewPageStateAction` is called, there's no need to `state.loaded = false` here because `fetchMonitorOverviewAction.get` does it already. 

There may be situations when  `fetchMonitorOverviewAction.get` may not be called when page state changes. So, setting `state.loaded = false` on page state change action was resulting in a bug, because there was no corresponding action which would set it back to `true`.

Reply:
The above change is not valid anymore as it turned out to be a different issue.